cmd/snap-exec/main.go

```diff
@@ -60,8 +65,8 @@ func run() error {
 	// confinement and (generally) can not talk to snapd
 	revision := os.Getenv("SNAP_REVISION")
 
-	snapApp := opts.Positional.SnapApp
-	return snapExec(snapApp, revision, opts.Command, args)
+	snapApp := args[0]
```

Comment:
Needs some basic validation I think. If args is empty we should have a nice error rather than a panic.